Namespace(add_gcn=False, alpha=1.0, batch_size=32, dataset='pubmed', device=0, dropout=0.5, early_stop=5, epochs=3000, hidden_dim=128, hidden_dim_dec_feat=128, hidden_dim_enc_adj=512, hidden_dim_enc_feat=512, lr=0.001, num_layers=1, seed=0, weight_decay=1e-05)
device :  cuda:0
Traceback (most recent call last):
  File "main.py", line 121, in <module>
    main()
  File "main.py", line 110, in main
    train(args, model, optimizer, adj, adj_norm, features)
  File "main.py", line 34, in train
    loss = total_loss(args, mu, var, adj_logits, feat_logits, adj, features)
  File "main.py", line 22, in total_loss
    adj_loss = args.alpha * F.binary_cross_entropy_with_logits(adj_logits, adj, pos_weight=pos_weight)
  File "/opt/anaconda3/lib/python3.6/site-packages/torch/nn/functional.py", line 2077, in binary_cross_entropy_with_logits
    return torch.binary_cross_entropy_with_logits(input, target, weight, pos_weight, reduction_enum)
RuntimeError: CUDA out of memory. Tried to allocate 1.45 GiB (GPU 0; 10.92 GiB total capacity; 9.03 GiB already allocated; 402.38 MiB free; 10.26 MiB cached)
